examples/torch/classification/main.py

```diff
@@ -266,7 +266,7 @@ def validate_fn(model, epoch):
             # training function that trains the model for one epoch (full training dataset pass)
             # it is assumed that all the NNCF-related methods are properly called inside of
             # this function (like e.g. the step and epoch_step methods of the compression scheduler)
-            def train_epoch_fn(compression_ctrl, model, epoch, optimizer, **kwargs):
+            def train_epoch_fn(compression_ctrl, model, optimizer, lr_scheduler, epoch):
```

Comment:
Why did you change signature of the `train_epoch_fn` function? 

The `lr_scheduler` is not used in the `train_epoch_fn` function. It looks like a bad style.

Reply:
The type hint for `train_epoch_fn` defined in the base `TrainingRunner` class is:
```
train_epoch_fn: Callable[[CompressionAlgorithmController, TModel,
                                        Optional[OptimizerType],
                                        Optional[LRSchedulerType],
                                        Optional[int]], None]
```

This didn't match signature of `train_epoch_fn` defined here and in other similar places. So I changed these definitions according to the type hint above (have just found another one for TF object detection which is not fixed yet, will do). 

Regarding the OTX, we are free to decorate train function as we desire so I see no conflict on that part.

Reply:
Regarding the bad style, one option is to replace unused arguments with `_`. On the other hand it would be less clear for the user in my opinion.

Reply:
Essentially, We need to understand where the typo was. If there is the typo in the type hint I will prefer to change type hint.

Reply:
Changed type hint